static/bidder-info/flipp.yaml

```diff
@@ -0,0 +1,7 @@
+endpoint: "https://cdn-gateflipp.flippback.com/flyer-locator-service/prebid_campaigns"
+maintainer:
+  email: prebid@flipp.com
```

Comment:
@hasan-kanjee , 

Thank you for your contribution to Prebid server. An email is sent from prebid team to verify the mentioned email address. Please respond back to email with "received" message to complete verification